docs/beta/grain.md

```diff
@@ -6,17 +6,34 @@ image: img/uploads/grain-token-icon.png
 ## What is Grain?
 
 Grain is a community-specific digital currency that is issued on the basis of
-[Cred scores](https://sourcecred.io/docs/beta/cred). Grain represents how much a
-participant has supported that community, either through contributions or
-financial support.
+[Cred scores](https://sourcecred.io/docs/beta/cred).  If cred answers the question
+"*who provided value*?", Grain answers ***"how should we reward people given the value
+they provided?"***
+
+### What is the value of Grain?
+Every community will use Grain differently.  In our community, Grain is currently
+redeemable for Dai, a cryptocurrency that is worth 1 US Dollar.  Communities without
+funds might treat this as a share of future income.  Another might treat it as weight
+behind a community member's vote.
+
+Note that Grain does not _have_ to be tied to any actual monetary reward. A project
+using SourceCred can name it “Points” and use it solely as a scoring system in addition
+to Cred values.
+
+In addition, communities do not have to distribute Grain at all.
```

Comment:
"In addition" was just used in the previous sentence; maybe use "Also" or something similarly different.

Reply:
Good point, done.